README.md

```diff
@@ -1,47 +1,71 @@
+$ npm rebuild node-sass  # this may be necessary
+$ npm start
+```
 
 ## Workflow
 
-- **npm start** <br> launch a demo server
-- **npm test** <br> run tests on ALL rhelements
-- **npm run build** <br> run build on ALL rhelements
-- **npm run bootstrap** <br> update ALL rhelements' dependencies and interlink them with [lerna bootstrap][lerna-bs]
+- `$ npm start` 
+    - Launch a demo server. This should be continuously running as you develop.
+- `$ npm run dev` 
+    -  Build a component.
+- `$ npm test` 
+    -  Run tests on ALL rhelements.
+- `$ npm run build` 
+    -  Run build on ALL rhelements.
+- `$ npm run bootstrap` 
+    - Update ALL rhelements' dependencies and interlink them with [lerna bootstrap][lerna-bs].
+- `$ npm run storybook`
+    - Run storybook 
 
 [lerna]: https://github.com/lerna/lerna
 [lerna-bs]: https://github.com/lerna/lerna#bootstrap
 
-## Component Development
+## Component development
 
-Each component that you work on will need to be built for you to see the changes. Every component has a gulpfile and an npm script, `npm run dev`, that you can run on each component that you're working on. The `npm run dev` command will watch the files in the component's directory and will run the build command when you make changes to the component.
+Because this is a monorepo, each component will need to be independently built in order to actively work on and preview the changes. Every component has its own Gulp file and NPM script. 
 
-### Example Development on a component
+While still running `npm start` in one terminal window (which runs the local server), you will need to open another terminal window, drill into the directory of the component you'd like to work on, and execute the `npm run dev` command. This command will use gulp tasks to watch the files within that component directory and will automatically re-run the build command and refresh the browser when you make changes to the component.
+
+### Example development on a component
 
 ```
-cd elements/rh-card
-npm run dev
+$ cd /Sites/rhelement
+$ npm start
+
+# SHIFT + CTRL + T to open a new tab in Terminal
+
+$ cd elements/rh-card  # or any other component
+$ npm run dev
 ```
 
-Make any changes you need and the gulpfile will handle transpiling the element down to ES5 and will bring in the HTML template and Sass file into the template in the component.
+Make a change to the component and save. The gulpfile will handle transpiling the element down to ES5 and will bring in the HTML and Sass into the template in the component.
 
-If you started the dev server at the root of RHElements, then you should be able to navigate to the demo page for the component, refresh the page, and see your changes.
+## Test
 
 To test all RHElements, run `npm test` from the root of the repo. If you only want to test the component you're working on:
 
-    cd elements/rh-card
-    npm test
+```
+$ cd elements/rh-card
+$ npm test               # DOESN'T WORK
```

Comment:
@starryeyez024 Odd.  When I run that, I see:

```

> @rhelements/rh-multiswitch@0.0.1 test /home/mclayton/projects/rhelements/elements/rh-multiswitch
> ../../node_modules/.bin/wct --configFile ../../wct.conf.json --npm node_modules/@rhelements/rh-multiswitch/test/

Installing and starting Selenium server for local browsers
Selenium server running on port 39819
chrome 68                Beginning tests via http://localhost:8081/components/@rhelements/rhelements/generated-index.html?cli_browser_id=0
chrome failed to maximize
chrome 68                Tests passed
Test run ended with great success

chrome 68 (1/0/0)                     
```

**Great** success!  It could be another one of the New Mac problems.  I'll see if @kylebuch8 is seeing the same thing.

Reply:
@mwcz I'm seeing the same thing.

Reply:
@starryeyez024 Run `npm install` from the root of the project and the testing should be fixed.